modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -0,0 +1,173 @@
+You need to specify the size of the image, in megabytes.
+Ensure you allow at least 300{nbsp}MB:
++
+----
+salt-call image_sync_usb.create <usb image name> <size in MB>
+----
+. Insert the USB device into the {smr} branch server machine, and copy the image to the new location:
++
+----
+dd if=<usb image name> of=<path to usb device>
+----
+
+
+When you have the image on the USB drive, check that the terminals you want to deploy allow local booting.
+You can check this by editing the Saltboot formula in the {smr} {webui}.
+For more information about the Saltboot formula, see xref:salt:formula-saltboot.adoc[].
+
+
+
+.Procedure: Deploying Images to the Terminals using USB
+. Insert the USB device into the terminal.
+. Power on the POS terminal.
+. Boot from the USB device to begin bootstrapping.
+
+
+
+== Bootstrap Terminals over a Wireless Network
+
+For terminals that cannot be connected directly to the physical network, you can bootstrap them over a wireless network.
+Wireless networks do not support PXE booting, so you must perform the initial booting and initialization of the wireless connection on the terminal using a USB device.
+
+For more information about using USB devices to boot, see xref:retail:retail-deploy-terminals-other.adoc[].
+
+
+[[WARNING]]
+====
+Bootstrapping across a wireless network could expose a security risk if you are using encrypted OS images.
+The boot ``initrd`` image and the partition that contains ``/etc/salt`` must be stored unencrypted on the terminal.
+This allows {smr} to set up the wireless network.
+If this breaches your security requirements, you will need to use a separate production network, with network credentials managed by Salt, so that credentials are not stored on the terminal unencrypted.
+====
+
+
+.Procedure: Preparing Wireless-Enabled Boot Image
+. Ensure that the ``dracut-wireless`` package is included  in the image template.
+. Provide the wireless credentials using the [command]``WIRELESS_ESSID`` and [command]``WIRELESS_WPA_PSK`` kernel options.
+. OPTIONAL: If you use the same wireless credentials for the entire organization, you can set them permanently by creating or editing the ``etc/sysconfig/network/ifcfg-wlan0`` file to the image template, with these details:
++
+----
+# ALLOW_UPDATE_FROM_INITRD
+WIRELESS_ESSID=<wireless network name>
+WIRELESS_WPA_PSK=<wireless network password>
+----
+If you want to use different credentials for bootstrapping to what is used during normal operation, you can remove the ``ALLOW_UPDATE_FROM_INITRD`` line.
+. Build the image.
+. Prepare a USB device using this image, and boot the terminal.
+For more information about using USB devices to boot, see xref:retail:retail-deploy-terminals-other.adoc[].
+
+Alternatively, you can create the file and add it to the ``initrd``:
++
+----
+echo ./etc/sysconfig/network/ifcfg-wlan0 | cpio -H newc -o | gzip >> /srv/saltboot/boot/initrd.gz
+----
```

Comment:
The thing is that this procedure is done on branch servers. I think it should be visibly separated from the instructions how to build wifi enabled image.